src/staticModelAdapt.cpp

```diff
@@ -118,7 +118,7 @@ namespace smc {
     int staticModelAdapt::calcMcmcRepeats(double acceptProb, double desiredAcceptProb, int initialN, int maxReps){
         if (acceptProb + 1.0 <= 1e-9){
             return initialN;
-        } else if (acceptProb - 1.0 <= 1e-9){
+        } else if (fabs(acceptProb - 1.0) <= 1e-9){
```

Comment:
I'm not sure what @eddelbuettel thinks about abs/fabs but I'd be inclined to just use std::abs in C++ code. Not that it is of any consequence... actually,on this instance I would have been inclined to say 
if ((acceptProb- 1.0) >= -1e-9) { ...
as there's not much danger of it being larger than 1+1e-9.